src/casscf/mcscf_2step.h

```diff
@@ -129,6 +129,11 @@ class MCSCF_2STEP {
     /// Orbital gradient convergence criteria
     double g_conv_;
 
+    /// Energy convergence for Davidson-Liu
+    double dl_e_conv_ = 1.0e-4;
```

Comment:
These are very loose defaults. Do you expect these would be useful or they always get override?

Reply:
These DL convergence thresholds will be tightened along the way through orbital optimization.

Reply:
Ok, it might be a good idea to say "These DL convergence thresholds will be tighten up along the way through orbital optimization" in the description of this variable.

Reply:
I changed these convergence thresholds to temporary variables before iterations.